### DISCUSSION TITLE
Is this "expected" after install ?

### DISCUSSION THREAD

**Discussion by miccerjatak** [matt8707/hass-config#80] [Q&A]
Tjena - i installed everything per "manual" - all good. But your design/ui components is not showing up at all - i only have a single page, with no sidebar etc. Are my HA crooked or did I miss something (probably the latter..). Any ideas to where to start solving this ? :-)

<img width="1953" alt="Skærmbillede 2022-09-03 kl  21 33 45" src="https://user-images.githubusercontent.com/112775618/188285439-d06057c0-f88f-4558-90d3-c4a491fc1e81.png">

**Comment by matt8707** [comment]
> Any ideas to where to start solving this ? :-)

yes, do ALL steps in [INSTALL.md](https://github.com/matt8707/hass-config/blob/master/INSTALL.md)
